docs/spec/bank/02_keepers.md

```diff
@@ -54,8 +54,8 @@ setCoins(addr AccAddress, amt Coins)
 
 `subtractCoins` fetches the coins of an account, subtracts the provided amount, and saves the account. This decreases the total supply.
 
-```
-subtractCoins(addr AccAddress, amt Coins)
+```go
```

Comment:
I don't think we need to modify any banking PRs here -- I think this will lead to a poor dev UX. Rather, simply modify the `handleMsgSend` and `handleMsgMultiSend` to check against the blacklist. Thoughts?

Reply:
true, true. That's a better solution